sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -635,6 +704,9 @@ from azure.core.credentials import AzureKeyCredential
 from azure.ai.documentintelligence import DocumentIntelligenceClient
 from azure.ai.documentintelligence.models import AnalyzeDocumentRequest, AnalyzeResult
 
+def format_price(price_dict):
+    return "".join([f"{p}" for p in price_dict.values()])
+
```

Comment:
Just curious, what does the output look like for this?

Reply:
The `price_dict` is a dict, for example: `{'currencySymbol': '$', 'amount': 999.0, 'currencyCode': 'USD'}`. So its formatted output would be `$999.0USD`.